RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -705,6 +715,47 @@ HcalNoiseInfoProducer::fillcalotwrs(edm::Event& iEvent, const edm::EventSetup& i
   return;
 }
 
+// ------------ fill the summary info from jets
+void
+HcalNoiseInfoProducer::filljetinfo(edm::Event& iEvent, const edm::EventSetup& iSetup, HcalNoiseSummary& summary) const
+{
+    bool goodJetFoundInLowBVRegion = false; // checks whether a jet is in
+                                            // a low BV region, where false
+                                            // noise flagging rate is higher.
+    if (!jetCollName_.empty())
+    {
+        edm::Handle<reco::PFJetCollection> pfjet_h;
+        iEvent.getByToken(jet_token_, pfjet_h);
+
+        if (pfjet_h.isValid())
```

Comment:
This is actually a direct translation of the previous code from
http://cmslxr.fnal.gov/source/CommonTools/RecoAlgos/plugins/HBHENoiseFilterResultProducer.cc?v=CMSSW_7_4_0_pre9#0136

Some workflows would, indeed, fail because they to not produce  ak4PFJets jet collection